src/ai/bot.js

```diff
@@ -211,11 +221,13 @@ export class MCTSBot extends Bot {
   playout(node) {
     let state = node.state;
 
-    for (
-      let i = 0;
-      i < this.playoutDepth && state.ctx.gameover === undefined;
-      i++
-    ) {
+    const playoutDepth = this.playoutDepth(
+      state.G,
+      state.ctx,
+      state.ctx.currentPlayer
```

Comment:
Why are we passing `currentPlayer` here? I would think that the playoutDepth shouldn't really depend on whose turn it is.  Also, you're passing `ctx` in, which contains this field anyway.

Reply:
I agree. I was only trying to keep the API consistent since it seems most other methods throughout the framework (including/especially the sibling ‘objectives’ method) seemed to have this signature.  I was curious about why player ID was added but I went along with it anyway.

Reply:
Did you want me to remove the `playerID` parameter, @nicolodavis?  I'm happy to do that.  But, if so, I'd also encourage we look at the other places where this is passed in unnecessarily.

Reply:
Yes, let's remove it for the code related to the present PR.

About the usage in other places, we only really need it to enumerate moves (which is why you see it in `createNode` and `playout`).  `objectives` are also player specific, which is why the `playerID` is passed there.  Let me know if there is any specific spot where you think it is redundant and we can revisit it.